src/appkit.cs

```diff
@@ -22888,5 +23434,68 @@ public interface NSWorkspaceAccessibilityExtensions {
 
 		[Export ("accessibilityDisplayShouldReduceTransparency")]
 		bool AccessibilityDisplayShouldReduceTransparency { get; }
+
+		[Mac (10, 12)]
+		[Export ("accessibilityDisplayShouldInvertColors")]
+		bool AccessibilityDisplayShouldInvertColors { get; }
+
+		[Mac (10, 12)]
+		[Export ("accessibilityDisplayShouldReduceMotion")]
+		bool AccessibilityDisplayShouldReduceMotion { get; }
+	}
+
+	[Mac (10,12)]
+	[BaseType (typeof(NSObject))]
+	interface NSFilePromiseProvider : NSPasteboardWriting
+	{
+		[Export ("fileType")]
+		string FileType { get; set; }
+
+		[Wrap ("WeakDelegate")]
+		[NullAllowed]
+		NSFilePromiseProviderDelegate Delegate { get; set; }
```

Comment:
note: it's not the only place that `NSFilePromiseProviderDelegate` is used. That's one of the downsides of `[Model]`, you don't want to force people into using them.